src/redux/data.js

```diff
@@ -904,7 +914,20 @@ const updatePurchases = updatedTransactions => dispatch => {
   dispatch(addCashUpdatePurchases(confirmedPurchases));
 };
 
-const watchPendingTransactions = (
+export const watchPendingTransactionsOnInitialize = (
```

Comment:
confirming if this is not going to be "watching" and just checking once on init, then can we rename to `watchPendingTransactionsOnInitialize`

Reply:
yes, only once